symbolic-demangle/src/lib.rs

```diff
@@ -32,6 +32,7 @@
 //! ```
 
 #![warn(missing_docs)]
+#![allow(clippy::return_self_not_must_use)]
```

Comment:
ugh, seems there's some clippy version arguments over this.  is there a way to disable that lint?

In any case, would it not make sense to put this on the actual functions which currently do this instead of global so to avoid introducing new ones?

Reply:
Oh, by "version argument" you are referring to the CI error?
```
error: unknown lint: `clippy::return_self_not_must_use`
```
That is indeed annoying. I removed this for now. Seems like we gotta have to go through a couple of updates anyway on thursday when the new rust update is there.

Reply:
> Oh, by "version argument" you are referring to the CI error?
> 
> ```
> error: unknown lint: `clippy::return_self_not_must_use`
> ```
> 
> That is indeed annoying. I removed this for now. Seems like we gotta have to go through a couple of updates anyway on thursday when the new rust update is there.

yeah, that's what i meant